### DISCUSSION TITLE
香橙派zero3怎么编译1.5G内存的固件

### DISCUSSION THREAD

**Discussion by waxll** [coolsnowwolf/lede#12104] [Q&A / 快速问答]
香橙派zero3默认编译的是2G内存的固件，我1.5G内存的板子无法启动

**Comment by aiamadeus** [comment]


**Reply by waxll** [reply]
我的好像不能正常识别，这个是tty的log
![Snipaste_2024-04-29_20-10-55](https://github.com/coolsnowwolf/lede/assets/63991291/033db437-de30-4a83-8fb6-5a6847722a52)
